cmd/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/neicnordic/sda-download/api"
+	"github.com/neicnordic/sda-download/internal/config"
+	"github.com/neicnordic/sda-download/internal/database"
+	"github.com/neicnordic/sda-download/internal/files"
+	"github.com/neicnordic/sda-download/internal/logging"
+	log "github.com/sirupsen/logrus"
+)
+
+// init is run before main, it loads the configuration variables
+func init() {
+	c := &config.Config
+	config.LoadConfig(c)
+	logging.LoggingSetup(c.App.LogLevel)
+	var err error
+	config.Config.App.Crypt4GHKey, err = files.GetC4GHKey()
+	if err != nil {
+		log.Errorf("failed to load Crypt4GH private key, re-encryption microservice required, %s", err)
+	}
```

Comment:
You can also initialise the error using `config.Config.App.Crypt4GHKey, err := files.GetC4GHKey()`

Reply:
The `err` can be initialised by inferring it, but because the `config.Config.App.Crypt4GHKey` already exists (we are placing the key into that location for later use), you can't create the variable because it already exists.

It's a case of
```
var a
a, b := func() // error
var a, b
a, b = func() // no error
```

Reply:
It could also have a different name. But it's ok.

Reply:
I could fix it with
```
key, err := files.GetC4GHKey()
config.Config.App.Crypt4GHKey = key
```

Reply:
I think both options are ok. This will likely be refactored when the configuration is redone.

Reply:
Ok, then I'll leave it for that